lib/tlGL/Render.cpp

```diff
@@ -716,15 +716,19 @@ namespace tl
             p.shaders["display"].reset();
         }
 
-        void Render::begin(const imaging::Size& size)
+        void Render::begin(const imaging::Size& size,
+                           const timeline::RenderOptions& renderOptions)
         {
             TLRENDER_P();
 
             p.size = size;
 
-            glViewport(0, 0, p.size.w, p.size.h);
-            glClearColor(0.F, 0.F, 0.F, 0.F);
-            glClear(GL_COLOR_BUFFER_BIT);
+            if ( renderOptions.clear )
+            {
+                glViewport(0, 0, p.size.w, p.size.h);
```

Comment:
I believe the "glViewport()" call should be left out of this conditional, it sets up the viewport for rendering.